src/helpers/3box.ts

```diff
@@ -1,14 +1,63 @@
 import { subgraphRequest } from '@snapshot-labs/snapshot.js/src/utils';
 
 export async function getProfiles(addresses) {
-  return subgraphRequest('https://api.3box.io/graph', {
-    profiles: {
-      __args: {
-        ids: addresses
-      },
-      name: true,
-      eth_address: true,
-      image: true
+  const [ensProfiles, threeBoxProfiles] = await Promise.all([
+    subgraphRequest('https://api.thegraph.com/subgraphs/name/ensdomains/ens', {

```

Comment:
Also it's better if we can separate the 2 methods. Can you create a method `lookupAddresses` that return an object with the ENS names:
```json
{
  "0x123": "fabien.eth",
  "0x345": "blabla.eth"
}
```

And a method `get3BoxProfiles` that return: 
```json
{
  "0x123": { "name": "Fabien" },
  "0x345": { "name": "Hello" }
}
```

And use them in the 
```js
const { ensNames, _3BoxProfiles } = await Promise.all[
  lookupAddresses(addresses),
  get3BoxProfiles(addresses)
]
```